src/transformers/utils/fx.py

```diff
@@ -978,24 +1148,39 @@ def trace(
             if param.kind == inspect.Parameter.VAR_KEYWORD and param.name not in input_names:
                 concrete_metas[f"**{param.name}"] = {}
         self.meta_args = concrete_metas
-        self.patched_torch_methods = {
-            target: _gen_constructor_wrapper(getattr(torch, target)) for target in self._TORCH_METHODS_TO_PATCH
+        self.orig_fns = set(getattr(torch, target) for target in self._TORCH_METHODS_TO_PATCH)
+
+        patched_torch_methods = {
+            f"torch.{target}": [
+                (torch, target, getattr(torch, target), _gen_constructor_wrapper(getattr(torch, target)))
```

Comment:
Isn't there redundancy in specifying the module, the attribute name, and the attribute?
With the (module, attr_name) we can always do `getattr(module, attr_name)` internally right?

Reply:
Yep you're right! This is done